Doppler.ContactPolicies.Business.Logic/Services/ContactPoliciesService.cs

```diff
@@ -21,5 +22,36 @@ public async Task<ContactPoliciesSettingsDto> GetContactPoliciesSettingsAsync(st
                 (await _contactPoliciesSettingsRepository.GetContactPoliciesSettingsAsync(accountName)).ToDto();
             return contactPoliciesSettings;
         }
+
+        public async Task<bool> InsertContactPoliciesSettingsAsync(string accountName, ContactPoliciesSettingsDto contactPoliciesSettings)
+        {
+            var existingUsercontactPolicies = await _contactPoliciesSettingsRepository.GetBasicContactPoliciesSettingsAsync(accountName);
```

Comment:
Very minor, but to continue following your own convention:

```suggestion
            var existingUserContactPolicies = await _contactPoliciesSettingsRepository.GetBasicContactPoliciesSettingsAsync(accountName);
```

Reply:
Fixed! marking as resolved...